openscap/openscap.go

```diff
@@ -0,0 +1,105 @@
+package openscap
+
+import (
+	"fmt"
+	"io"
+	"net/http"
+	"net/url"
+	"os"
+	"os/exec"
+	"path"
+	"strings"
+)
+
+const (
+	CPE               = "oval:org.open-scap.cpe.rhel:def:"
+	MAX_DIST          = 7
+	MIN_DIST          = 5
```

Comment:
@enoodle I think there's a way to create a static array, something like `[]int {5, 6, 7}`.